content/reference/current/_index.md

```diff
@@ -4,8 +4,8 @@ description: "Gatling Enterprise Self-Hosted documentation"
 cascade:
   variables:
     revnumber: "1.18.0"
-    ciPluginsVersion: "1.16.4"
-    grafanaPluginVersion: "1.16.4"
+    ciPluginsVersion: "1.16.5"
+    grafanaPluginVersion: "1.16.5"
```

Comment:
Unrelated, but we published version 1.16.5 of the Grafana plugin a long while ago...

Reply:
Ouch